lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -391,6 +393,7 @@ sub template_data_from_yaml {
                 my $prio          = $yaml_defaults_for_arch->{priority};
                 my $machine_names = $yaml_defaults_for_arch->{machine};
                 my $settings      = dclone($yaml_defaults_for_arch->{settings} // {});
+                my $description   = $yaml_defaults_for_arch->{description} // '';
```

Comment:
Two questions:
1. I don't see this in the schema
2. When is having several scenarios with the same description useful?

Reply:
Oh right, I think I left that out of the schema because of 2.

Reply:
replaced it with `my $description   = '';`

Reply:
Thanks!